.github/workflows/build-and-push-components-manual.yml

```diff
@@ -0,0 +1,14 @@
+name: Build and Push Components Manual
+
+on:
+  workflow_dispatch:
```

Comment:
just checking here if we did any tests that we can check for this step. would be good to include those details here. im not sure if this will work as intended, so please share more info here

Reply:
I've collected details on the recent changes here:

https://github.com/openclarity/vmclarity/pull/1848#discussion_r1657030770